src/pages/resultsView/comparison/ComparisonTab.tsx

```diff
@@ -107,6 +114,14 @@ export default class ComparisonTab extends React.Component<
         },
     });
 
+    @computed get alterationEnrichmentTabName() {
+        const nameElements = [];
```

Comment:
might as well make this an export function (that takes store as arg) so we don't have to duplicate this logic

Reply:
Done. I created the function in ComparisonStoreUtils.ts.